augur/io/sequences.py

```diff
@@ -1,29 +1,39 @@
 import Bio.SeqIO
 
 from augur.errors import AugurError
+from typing import Iterator, Iterable
 from .file import open_file
 
 
-def read_sequences(*paths, format="fasta"):
+def read_sequence(
+    path: str,
+    format: str = "fasta",
+) -> Bio.SeqIO.SeqRecord:
+    """Read a single sequence from a path.
+
+    Automatically infers compression mode.
+    """
+    with open_file(path) as handle:
+        return Bio.SeqIO.read(handle, format)
+
+
+def read_sequences(
+    *paths: Iterable[str],
```

Comment:
You're narrowing the type here vs the previous docstring - can you explain why you dropped `os.PathLike`? This might be on purpose, just lacking context!